src/css/_components.css

```diff
@@ -10,3 +11,19 @@
   margin: 0 0 20px;
 
 }
+.site-header{
```

Comment:
Maybe consider adding these styles to `_header.css` as they aren't really shared styles